public/Day-217/styles/circuit.css

```
@@ -0,0 +1,201 @@
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: var(--bg-dark);
+    color: var(--text-main);
+    font-family: 'Inter', sans-serif;
```

Stylelint (17.2.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)